examples/ui/client-capabilities/interaction/text-field.js

```diff
@@ -0,0 +1,51 @@
+import { CodeSection } from '../../documentationBase';
+import useIsBrowser from '@docusaurus/useIsBrowser';
+
+export default function TextFieldDemo({ children, color }) {
+
+	const isBrowser = useIsBrowser();
+
+	if (isBrowser) {
+		const RapidImports = require('../../rapidImports');
+		RapidImports.registerComponents();
+	}
+
+	return (
+		<CodeSection>
+
+
+			<div style={{ color: 'var(--neutral-foreground-rest)', width: '-webkit-fill-available'}}>
+				<div style={{ display: 'flex', flexWrap: 'wrap', gap: '10px', flexDirection: 'column', }}>
+
+					<rapid-text-field placeholder="Sample text" type={"text"}>Text Field</rapid-text-field>
+
+					<rapid-text-field placeholder="Format: .+@example\.com" type={"email"} pattern=".+@example\.com">Email Text Field</rapid-text-field>
```

Comment:
the `type` attributes don't need any special syntax as they are just static strings - can just be `type="email|password|tel" etc...